src/index.js

```diff
@@ -26,20 +28,21 @@ class TwitterLogin extends Component {
     }).then(response => {
       return response.json();
     }).then(data => {
-      this.openPopup(data.oauth_token);
+      popup.location = 'https://api.twitter.com/oauth/authenticate?oauth_token=' + data.oauth_token;
```

Comment:
Can you please use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for string creation. I think that it is much easier to read